prettyprinter/test/Testsuite/Main.hs

```diff
@@ -1,6 +1,6 @@
 {-# LANGUAGE CPP               #-}
 {-# LANGUAGE OverloadedStrings #-}
-{-# OPTIONS_GHC -Wno-orphans   #-}
+{-# OPTIONS_GHC -fno-warn-orphans   #-}
```

Comment:
I didn't originally want to include this here. But it allows building the tests with GHC 7.8 and 7.10…